examples/00-mapdl-examples/composite_dcb.py

```diff
@@ -0,0 +1,415 @@
+"""
+.. _ref_dcb_test_composite_delamination
+
+======================================================================
+Static simulation of double cantilever beam test via cohesive elements
+======================================================================
+
+This example is inspired to a classic double cantilever beam test commonly used
+to study mode I interfacial delamination of composite plates.
+
+Description
+===========
+
+Objective
+~~~~~~~~~
+
+This example shows how to use PyMAPDL to simulate delamination in
+composite materials. PyDPF modules are also used for the post-processing of the results.
+
+Problem Figure
+~~~~~~~~~~~~~~
+.. image:: ../../../images/dcb_test.png
+   :width: 400
+   :alt: DCB experimental test setup
+
+Procedure
+~~~~~~~~~
+* Launch MAPDL instance
+* Setup the model
+* Solve the model
+* Plot results of interest using PyMAPDL
+* Plot results using PyDPF
+* Plot reaction force
```

Comment:
```suggestion
* Launch MAPDL instance
* Set up the model.
* Solve the model.
* Plot results using PyMAPDL.
* Plot results using PyDPF.
* Plot reaction force.
```

Reply:
Google Developer's Style Guide says that if bulleted list items have a given subject (You) and a verb to conclude them with periods.